extension/js/background.js

```diff
@@ -41,9 +41,17 @@ async function getOrMakeGAUUID() {
 
 
 async function sendMetricsEvent(eventData) {
-  if (navigator.doNotTrack === "1") {
+  const doNotTrackIsEnabled = (navigator.doNotTrack === "1");
+  const { dataCollection } = await browser.storage.local.get("dataCollection");
+
+  if (!dataCollection) {
+    browser.storage.local.set({ "dataCollection": "data-enabled" });
+  }
```

Comment:
This means opt-out, so we'll have to make sure the approved content also indicates opt-out.